src/components/navigator/util.tsx

```diff
@@ -0,0 +1,26 @@
+import truncate from 'lodash.truncate';
+
+import { Link } from 'react-router-dom';
+
+import { Typography, styled } from '@mui/material';
+
+import { ITEM_NAME_MAX_LENGTH } from '../../config/constants';
+import { buildItemPath } from '../../config/paths';
+
+const StyledLink = styled(Link)({
+  textDecoration: 'none',
+  color: '#000000',
```

Comment:
Why did you need to use black? Was the link blue? Try to use the `theme` color instead. I think `text.primary` might work? https://mui.com/system/palette/